### PR TITLE
Kube pod fix

### PR DESCRIPTION
**Pull request by dbrinegar**
fixes lightbend/es-backend#238

### CODE REVIEW

File: enterprise-suite/es-monitor-api/static-rules.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@
   expr: kube_pod_status_phase{phase="Failed"} * on (pod) group_left(workload) kube_pod_info
 
 - record: kube_pod_not_running
-  expr: kube_pod_status_phase{phase=~"Pending|Unknown"} * on (pod) group_left(workload) kube_pod_info
+  expr: max without (phase) (kube_pod_status_phase{phase=~"Pending|Unknown"} * on (pod) group_left(workload) kube_pod_info)
```
Comment (jsravn) [inline enterprise-suite/es-monitor-api/static-rules.yml]: Should we consider `Failed` phase as well?

Comment (dbrinegar) [reply enterprise-suite/es-monitor-api/static-rules.yml]: I _think_ that Failed is when Kubernetes is not trying to run the pod anymore, and will trigger other alerts.  This one is perhaps not named clearly, but is trying to identify things that Kubernetes is trying to run but aren't running yet.  So Kubernetes is spinning away retrying or waiting for something, the alert lets you put an SLA on that so you can be alerted if things don't get going in X minutes.

Comment (dbrinegar) [reply enterprise-suite/es-monitor-api/static-rules.yml]: But yeah there's a lot of field testing and validation needed on these, much of this is conceptual or borrowed, not based on in-house rules we know work well.

Comment (dbrinegar) [reply enterprise-suite/es-monitor-api/static-rules.yml]: "kube_pod_hung_in_limbo" ?

**Pull request merged**